indra/sources/rlimsp/api.py

```diff
@@ -105,10 +105,23 @@ def process_from_jsonish_str(jsonish_str, doc_id_type=None):
         An RlimspProcessor which contains a list of extracted INDRA Statements
         in its statements attribute.
     """
-    lines = jsonish_str.splitlines()
-    json_list = []
-    for line in lines:
-        json_list.append(json.loads(line))
+    json_list = [json.loads(line) for line in jsonl_str.splitlines()]
     rp = RlimspProcessor(json_list, doc_id_type=doc_id_type)
     rp.extract_statements()
     return rp
+
+
+# DEPRECATED functions
+
+def process_from_json_file(filename, doc_id_type=None):
+    """DEPRECATED: use process_jsonl_file instead."""
+    logger.warning('process_from_json_file is deprecated. Use '
```

Comment:
there's a module in python called `warn` that you can use like

```suggestion
    warn.warning('process_from_json_file is deprecated. Use process_jsonl_file instead.', DeprecationWarning)
```